### DISCUSSION TITLE
Connection closed error

### DISCUSSION THREAD

**Discussion by hunhejj** [nextcloud/all-in-one#4821] [Questions] [answered]
We are trying to do the initial synchronization for a new user. The desktop client reports that many files can't be uploaded ("Connection closed" error).

AFAICT it mainly affects files with a size of at least 500-600 MB. I didn't override `NEXTCLOUD_MAX_TIME` and `NEXTCLOUD_UPLOAD_LIMIT` when setting up AIO.

When checking the logs of the _apache_ container, I see a lot of these entries:
> [error] [(70008)Partial results are valid but processing is incomplete] [client: 84.2.26.154, 172.22.0.1] [AH01075: Error dispatching request to : (reading input brigade)] [Mozilla/5.0 (Windows) mirall/3.13.0stable-Win64 (build 20240423) (Nextcloud, windows-10.0.19045 ClientArchitecture: x86_64 OsArchitecture: x86_64)]

Any ideas what is causing this?

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this question at https://help.nextcloud.com thanks!
